### ISSUE TITLE
upgrade to spring 4

### ISSUE THREAD

**Issue by lkwg82ATis24** [connect-group/thymeleaf-tdd#2]
Hi,

we use spring 4 in our current project and would like to use your testing lib. 

How much do you estimate is the effort to replace spring3 with spring4?

Cheers Lars

**Comment by 4dz** [comment]
Haven't tried yet but I'll give it a go and let you know!

**Comment by 4dz** [comment]
The answer is "yes" it does work with Spring 4. I need to add a few projects into the maven repository, (takes a while). I've committed a spring4 branch in thymeleaf-tdd; created a thymesheet-spring4 project; and I'm updating the thymesheet-sample project to use Spring 4.

It wasn't much effort, mostly renaming a few packages and updating some pom's.

**Comment by 4dz** [comment]
I've released thymeleaf-tdd-spring4 to Maven, it will take a few hours to appear (I think up to 8 hours typically).  All the source code is here on GitHub, though, if you want to see it working sooner.

I've added a new thymesheet-spring4 project.

The thymesheet-sample project has now been updated to use Spring4 and has some Junit/Thymeleaf tests added.  Hopefully some of this will be useful to you?

All the best


Adam
